app/services/api/teachers/school_transfers/query.rb

```diff
@@ -77,7 +77,18 @@ def where_lead_provider_is(lead_provider_id)
     def where_updated_since(updated_since)
       return if ignore?(filter: updated_since)
 
-      @scope = scope.where(api_updated_at: updated_since..)
+      @scope = scope
+          .left_joins(
+            ect_at_school_periods: %i[earliest_training_period latest_training_period],
+            mentor_at_school_periods: %i[earliest_training_period latest_training_period]
+          )
+          .where(
+            "(training_periods.id IS NOT NULL AND training_periods.api_transfer_updated_at >= :updated_since) OR
+            (latest_training_periods_ect_at_school_periods.id IS NOT NULL AND latest_training_periods_ect_at_school_periods.api_transfer_updated_at >= :updated_since) OR
+            (earliest_training_periods_mentor_at_school_periods.id IS NOT NULL AND earliest_training_periods_mentor_at_school_periods.api_transfer_updated_at >= :updated_since) OR
+            (latest_training_periods_mentor_at_school_periods.id IS NOT NULL AND latest_training_periods_mentor_at_school_periods.api_transfer_updated_at >= :updated_since)",
+            updated_since:
+          )
```

Comment:
Should we add some tests for the filter applying when the school periods have been updated since a given date?

Reply:
Yeah good shout I'll add more explicit tests around this filtering 👍